pkg/issue/issue.go

```diff
@@ -92,8 +95,12 @@ func ensure(ctx context.Context, c *github.Client, issues issues, owner, repo, p
 	if err != nil {
 		return err
 	}
-	if issue == nil {
-		oc, _, _ := configGetAppConfigs(ctx, c, owner, repo)
+	now := timeNow()
```

Comment:
Actually, if we are only using now here to pass into `schedule.ShouldPerform`, just remove the time as a parameter to `schedule.ShouldPerform` and have it call timeNow() directly (mocked for testing).

Reply:
Package issue shouldn't worry about the time, it should just be unit tested to call `schedule.ShouldPerform` and only create/ping if that returns true.